_posts/2021-04-27-Optimising-ux-for-software.markdown

```diff
@@ -7,70 +7,65 @@ author: Lucy Sharp
+I wanted to better understand this relationship, considering DevOps is Mechanical Rock's way of working. So I started researching and listening to people. Many coffees later, it became apparent that value is realised when we bring UX into DevOps ways of working. Such optimisiation enables flow through a system which accelerates delivery and provides a competitive advantage.
+
+I've put together three reasons why you should optimise UX for Software.
 <br /><br />
 
-## What is UX?
+## Let's start by understanding what UX is...
 
 User Experience (UX) “encompasses all aspects of the end user's interaction with the company, its services, and its products” ([Human Centred Design in QLD Toolkit](https://www.forgov.qld.gov.au/human-centred-design-resources)).
 
-UX provides insight into the gaps, challenges and opportunities of a problem space within a specific domain to improve user experience and deliver a specific ROI to key stakeholders and customers. For every one dollar invested in UX yields a hundred dollar return; a return of 9,900% as reported by [Forrester](https://www.forrester.com/report/The+Six+Steps+For+Justifying+Better+UX/-/E-RES117708).
+UX provides insight into the gaps, challenges and opportunities of a problem space within a specific domain to improve user experience and deliver a specific return of investment (ROI) to key stakeholders and customers. For every one dollar invested in UX yields a hundred dollar return; a return of 9,900% as reported by [Forrester](https://www.forrester.com/report/The+Six+Steps+For+Justifying+Better+UX/-/E-RES117708).
 
-Unfortunately, the inherent value of UX is often not recognised and this may reflect designers ways of working. Traditionally, design operates in a more waterfall environment where siloed work coupled with a lack of understanding about how the system operates impacts user experience and leads to design re-work or developers making decisions without a UX designer ([Talia Lavie](https://techbeacon.com/devops/8-ways-integrate-ux-speed-devops)). Such siloed work between designers and developers is a constraint; slowing down development and delivery. With the evolution of processes and culture, UX practices need to change. A driver of this evolution is DevOps.
+Unfortunately, the inherent value of UX is often not recognised and this may reflect designers ways of working. Traditionally, design operates in a more waterfall environment where siloed work coupled with a lack of understanding about how the system operates, impacts user experience and leads to design re-work or developers making decisions without a UX designer ([Talia Lavie](https://techbeacon.com/devops/8-ways-integrate-ux-speed-devops)). Such siloed work between designers and developers is a constraint; slowing down development and delivery. With the evolution of processes and culture, UX practices need to change. A driver of this evolution is DevOps.
 <br /><br />
 
-## DevOps
+## Wait, we need to know what DevOps is too...
```

Comment:
Same as above - as a reader I've lost the thread.

`What is DevOps?` would be an easier title to read.

Don't make me think.